### ISSUE TITLE
There is no function to remove server logs on aria-role-log/serverlog.html working-example

### ISSUE THREAD

**Issue by mehm8128** [w3c/wcag#4993]
I tried `serverlog.html` working-example and confirmed that the log addition are announced by AT correctly, but I was not able to confirm that the removal of the log is not announced, because there is no button to remove and also auto removal due to the log size limitation is not performed.

https://www.w3.org/WAI/WCAG22/Techniques/aria/ARIA23
https://www.w3.org/WAI/WCAG22/working-examples/aria-role-log/serverlog.html

It looks that the algorithm to remove logs is commented out, so when we uncomment it and make some fix to work correctly, we'll be able to confirm that logs are correctly removed (and I'm happy to fix it if it's OK).

https://github.com/w3c/wcag/blob/c9c2c61c4d4df8d19544b673a541b95dc9026ce2/working-examples/aria-role-log/serverlog.html

**Comment by patrickhlauke** [comment]
Feel free to submit a PR if you can
